question-service/router/router.js

```diff
@@ -0,0 +1,29 @@
+import express from "express";
+import { createQuestion, getAllQuestions, getQuestionById, deleteQuestionById, updateQuestionById, getFilteredQuestions, findQuestion } from "../manager/questionManager.js";
+import validateQuestion from '../middlewares/validation.js';
+
+const router = express.Router();
+
+// CREATE NEW QUESTION
+router.post('/', validateQuestion, createQuestion);
+
+// GET ALL QUESTIONS
+router.route("/").get(getAllQuestions);
+
+// GET QUESTION BY ID
+router.route("/id/:id").get(getQuestionById);
+
+// DELETE QUESTION BY ID
+router.route("/id/:id").delete(deleteQuestionById);
+
+// UPDATE QUESTION BY ID
+router.route("/id/:id").put(updateQuestionById);
```

Comment:
should create, delete and update be accessible only to admins? (while get is anyone)